CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 3.7.0 (not yet released)
+
+* Added support for `datadog` handler
+* Configure the `DatadogFormatter` as a service
```

Comment:
I don't think we cares about `DatadogFormatter` being available in the Symfony container, since use wrappers and not formatters directly. WDYT?

Reply:
It is configured only if you have a `datadog` type. 
see:  https://github.com/symfony/monolog-bundle/pull/364/files#diff-25076ecfab002a680c691e32a36c5e6bR899

Reply:
Yes I know, but the user does not have to directly deal with handlers/formatters, so I don't think this changelog entry is required... but I may be wrong.

Reply:
I agree with @Kocal. Could you remove this line?
And could you also rebase?
Thanks